test/e2e/suite/leaderelection/leaderelection.go

```diff
@@ -29,82 +25,7 @@ var _ = framework.CasesDescribe("leader election", func() {
 		By("Waiting for all pods to be ready")
 		err := f.Helper().WaitForPodsReady(f.Config().Namespace, 10*time.Second)
 		Expect(err).NotTo(HaveOccurred())
-		By("Finding all cas issuer leader election config maps")
```

Comment:
Since updating controller-runtime, google-cas-issuer now uses lease resources for leader election. Here we remove these tests revolving around ConfigMaps. Not sure these tests are needed at all to be honest.